blob/commitment_proof.go

```diff
@@ -4,9 +4,9 @@ import (
 	"bytes"
 	"fmt"
 
-	"github.com/celestiaorg/celestia-app/pkg/appconsts"
-	"github.com/celestiaorg/celestia-app/pkg/shares"
```

Comment:
[note for reviewers] I'm not sure how this slipped through https://github.com/celestiaorg/celestia-node/pull/3453. Maybe because this file was created after that PR first started. Regardless, this repo should no longer import from `github.com/celestiaorg/celestia-app` and should instead import from `github.com/celestiaorg/celestia-app/v2`.